camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/dsl/methods/DslComponent.java

```diff
@@ -410,6 +424,7 @@ public String toString() {
         }
     }
 
+    @DslAccessible
```

Comment:
(does the comment on the PR header, "DslComponent is a special case where it will not be resolved before evaluating the DSL as that's the expected behaviour.", still apply?)